src/elements/content-sidebar/withSidebarAnnotations.js

```diff
@@ -0,0 +1,179 @@
+// @flow
```

Comment:
Should this be a TS file?

Reply:
I gave it a shot, but it looks like there is a lot of work to get the current flow types converted over to TS, can take a look in a follow up PR as well